dotCMS/src/main/java/com/dotcms/workflow/form/FireActionForm.java

```diff
@@ -95,6 +101,8 @@ public FireActionForm(final Builder builder) {
         private String whereToSend;
         @JsonProperty()
         private String forcePush;
+        @JsonProperty()
+        private String filterKey;
```

Comment:
Codacy found an issue: [Field filterKey has the same name as a method](https://app.codacy.com/manual/dotCMS/core/pullRequest?prid=5919676)